### DISCUSSION TITLE
Change path of terminal to currently focused file

### DISCUSSION THREAD

**Discussion by johan-von-essen** [zed-industries/zed#20983] [Feature Requests]
There are some instructions on how to copy currently marked text and copy/paste it to the terminal.
See https://github.com/zed-industries/zed/discussions/8674#discussioncomment-11170937 for such an example.

I would like to focus the terminal and go the folder of the file currently focused. However, I cannot figure out how to grab the path before starting to paste it into the terminal. Halp?!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
